api/tests/middleware/authorization-middleware.test.ts

```diff
@@ -0,0 +1,83 @@
+import { Response, NextFunction } from "express"
+import { User } from "@/models"
+import { auth0Integration, type Auth0UserInfo } from "@/integrations/auth0-integration"
+import {
+  authorizationMiddleware,
+  type AuthorizationRequest,
+} from "@/middleware/authorization-middleware"
+
+jest.unmock("@/middleware/authorization-middleware")
+
+jest.mock("@/integrations/auth0-integration")
+jest.mock("@/utils/logger")
+
+let user_data: Auth0UserInfo
+let req: Partial<AuthorizationRequest>
+let res: Partial<Response>
+let next: NextFunction
+
+beforeEach(() => {
+  user_data = {
+    email: "dupe@test.com",
+    firstName: "dupe",
+    lastName: "UNKNOWN",
+    auth0Subject: "",
+  }
+
+  req = {
+    headers: {
+      authorization: "mock-token",
+    },
+    auth: {},
+  }
+
+  res = {
+    status: jest.fn().mockReturnThis(),
+    json: jest.fn(),
+  }
+
+  next = jest.fn()
```

Comment:
Generally this kind of setup would go inside the inner most describe block that it effects.